CHANGELOG.md

```diff
@@ -1,3 +1,214 @@
+GOBBLIN 0.8.0
+-------------
+
+#### Created Date: 08/22/2016
+
+## Highlights
+
+* Gobblin can now convert avro to orc files. 
+* Gobblin can now support hive based replication.
+* Gobblin can now support hive based retention.
+* Gobblin can now support job templates, which reduces the efforts of writing a Gobblin job.
+    Documentation: http://gobblin.readthedocs.io/en/latest/user-guide/Gobblin-template/
+
+## NEW FEATURES
+
+* [Kafka] [PR 1016] Integration with Confluent Schema Registry, Confluent Deserializers, and Kafka Deserializers
+* [Avro to ORC] [PR 1031] Adding Avro To ORC conversion logic and related framework modifications
+* [General FileSystem Support] [PR 1066] Config file monitor for general file system
+* [Avro to ORC] [PR 1068] Nested Avro to Nested ORC conversion support
+* [General FileSystem Support] [PR 1073] extension of loading config file from general file system
+* [AWS] [PR 1088] Gobblin on AWS
+* [Kafka Writer] [PR 1089] Kafka writer
+* [JDBC Extractor] [PR 1090] Teradata JDBC Extractor and Source
+* [Avro to ORC] [PR 1093] Support for schema evolution, staging, selective column projection and compatibility check for Avro to ORC
+* [Hive Retention] [PR 1106] Hive Based Retention
+* [Job Launcher] [PR 1130] Initial commit for new gobblin launch interfaces
+* [Job Launcher] [PR 1131] Gobblin launcher standard implementations
+* [Job Launcher] [PR 1133] Adding caching decorators for JobCatalog
+* [Job Templates] [PR 1145] Initial commit for job configuration template
+* [Job Launcher] [PR 1164] JobLauncher interface update
+* [Job Launcher] [PR 1168] Initial commit for a skeleton GobblinInstanceDriver
+* [Job Launcher] [PR 1174] Base implementation for Kafka job spec monitor.
+* [Job Launcher] [PR 1178] More changes toward an end-to-end flow with the new APIS
+* [Avro to ORC] [PR 1188] Avro to orc data validation
+* [Job Launcher] [PR 1194] More changes to Gobblin Launcher API
+* [Job Templates] [PR 1197] Kafka-template
+* [Job Launcher] [PR 1203] New std driver2
+* [Core] [PR 1216] Adding a simple console writer to gobblin
+
+## BUG FIXES
+
+* [Build] [PR 977] Reverting FindBugs fix for JdbcExtractor
+* [YARN] [PR 982] Using new zk port numbers for unit tests
+* [Build] [PR 984] Fixing new FindBugs warnings, and setting Travis build to fail if new FindBugs warnings are added
+* [Kafka] [PR 996] Fix offset related bug in KafkaSource
+* [Core] [PR 999] distcp-ng throws UnsupportedOperationException
+* [Build] [PR 1000] Adding missing testCompileDependency
+* [Build] [PR 1001] Setting heaps size for gobblin-runtime tests due to OOM in some cases
+* [Core] [PR 1002] Set explicit 755 permissions to state store
+* [Core] [PR 1005] Fixing SOURCE_QUERYBASED_LOW_WATERMARK_BACKUP_SECS no default value
+* [Hive Distcp] [PR 1015] Fix hanging IteratorExecutor. HiveDataset will skip bad tables instead of failing the entire job.
```

Comment:
Since the hive replication is in the highlights, we don't have to include all the PRs about it. Same for other highlights.